pkgs/development/libraries/mesa/default.nix

```diff
@@ -235,7 +235,7 @@ self = stdenv.mkDerivation {
     done
 
     # NAR doesn't support hard links, so convert them to symlinks.
-    ${jdupes}/bin/jdupes -Hlr "$drivers"
+    ${jdupes}/bin/jdupes --hard-links --link-soft --recurse "$drivers"
```

Comment:
Wouldn't this comment be lying now?

Reply:
I don't get what you mean.  This diff only converted short options to the same long ones.

Reply:
Perhaps the confusion here is that the point of `--hard-links` is to *read* the hard links generated by the Mesa build system, and treat them as duplicates to be converted to symlinks; it does not *create* hard links (that would be `--link-hard`).

Reply:
@andersk bingo. Assumptions assumptions. Thanks.